### ISSUE TITLE
Breadcrumb Component A11Y Issues

### ISSUE THREAD

**Issue by mumenthalers** [oblique-bit/oblique#134]
### Oblique version
11.3.2

### Description

Following aria attributes are currently missing but should be used to be understood when using assistive technology:

- `aria-label="Breadcrumb"` on the `nav` tag to label the landmark\
https://github.com/oblique-bit/oblique/blob/d9ed7d79042f407b61f355b21039cbde8dc4829a/projects/oblique/src/lib/breadcrumb/breadcrumb.component.html#L1

- `aria-current="page"` on the lists last element (`span`)\
https://github.com/oblique-bit/oblique/blob/d9ed7d79042f407b61f355b21039cbde8dc4829a/projects/oblique/src/lib/breadcrumb/breadcrumb.component.html#L18

Another issue is caused by the usage of the material `Tooltip` which gets enabled when label exceeds the max width and is cut off.
The material tooltip adds the provided text as `aria-description` (even if disabled). Since the tooltip text is the same as the actual link text, some screen readers (tested with VoiceOver on MacOS) will end up reading out the label twice for each link.
This should be fixable by one of the following options:
- stop using the material tooltip
- add the link text as `aria-label` on the same element as the tooltip. This may sound counterproductive but has 2 effects: *1st* the aria-label text will override the text supplied within the link. *2nd* the cdk `AriaDescriber` (used by the matTooltip) will not add the `aria-description` since [it checks whether there's already an `aria-label` on the element with the same value.](https://github.com/angular/components/blob/main/src/cdk/a11y/aria-describer/aria-describer.ts#L286)

### Reproduction
[render the component](https://stackblitz.com/edit/oblique-breadcrumb?file=src%2Fapp%2Fbreadcrumb-sample%2Fbreadcrumb-sample.component.html) and 
- visualize the Landmarks 
- use a screenreader and navigate to the breadcrumb links
with e.g. VoiceOver on Mac

### Expected behaviour
- when listing the landmarks the breadcrumb navigation should be labeled as breadcrumb navigation
- when using VoiceOver on Mac the Links are not read twice
- the last item in the breadcrumb list is mentioned as current page from the screen reader

### Context
- [MDN Breadcrumb Navigation](https://developer.mozilla.org/en-US/docs/Web/CSS/Layout_cookbook/Breadcrumb_Navigation)

**Comment by gillerr** [comment]
Hi @mumenthalers 

Thank you for your report, we'll fix that as you described.

**Comment by nina-egger** [comment]
Fixed with 12.0.1
